homework/index.js

```diff
@@ -30,15 +15,162 @@
     return elem;
   }
 
+  function remove() {
+    document.getElementById('right_column_list').remove();
+  }
+
+  function setLink(linkID, srcArray) {
+    document.getElementById(linkID).innerText = srcArray.name;
```

Comment:
If you look closely, you are trying to fetch the element here in all three steps before you assign its `innerText`, `href` and `target` attributes. You can instead have the calling function i.e from where we call `setLink` to provide the element itself or simply retrieve it once in this function and use it in all three assignments. This way, you wouldn't have to fetch the same element thrice and will save yourself two operations. 

```
function setLink(linkId, srcArray) {
  const elem = document.getElementById(linkID);
  elem.innerText = srcArray.name;
  elem.setAttribute('href', srcArray.html_url);
  elem.setAttribute('target', '_blank');
}
```

Reply:
Replaced your code but I understand what you meant.
Surely, this should be like this.